packages/inter-protocol/src/price/README.md

```diff
@@ -4,6 +4,19 @@ This directory contains the `fluxAggregatorKit.js` contract which takes prices a
 input and outputs a best known price. There are multiple ways to get the price,
 including a PriceAuthority interface.
 
+## Design
+
+The authorities are hierarchical. Many authorities can be registered in the priceAuthorityRegistry.
+
+For oracles, there must be two: the negotiable brand and the _oracle_ brand (an inert one).
+
+The intended flow is that:
+1. a negotiable brand is created (e.g. ATOM)
+2. a price provider says “i can give you quotes for that” and runs price-feed-proposal. That makes “oracleBrands” (which are inert and have a separate identity so that they don’t have the authority to say they’re the real quote for it).
```

Comment:
I presume 'inert' means no payments or purses. If that's the case, please say so explicitly. Is there any way for holders of the issuer or the brand to reliably detect that it is inert?

Reply:
Right, there's no mint or issuer. But there's no way to inspect the brand to find out.